docs/root/configuration/http/http_filters/proto_message_extraction_filter.rst

```diff
@@ -1,23 +1,23 @@
 .. _config_http_filters_proto_message_extraction:
 
-Proto Message Extraction
+Proto message extraction
```

Comment:
not sure what is best here - we dont capitalize headings, altho we do for names/proper nouns

perhaps original was correct and the name _is_ "Proto Message Extraction" - but in that case i think it should be updated below where the ~code name - ie `ProtoMessageExtraction` is used

Reply:
That's actually the filter name. 

IMO, we could keep it as `Proto Message Extraction` filter at both places.

Thanks!